packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataOutlineProvider.test.ts

```diff
@@ -205,24 +391,65 @@ describe('load org browser tree outline', () => {
     loadCmpsStub.restore();
   });
 
+  it('should check for parent node when a folder node is selected', async () => {
+    const expected = [
+      {
+        label: 'Id (id)',
+        fullName: 'Id (id)',
+        type: NodeType.MetadataField
+      },
+      {
+        label: 'IsDeleted (boolean)',
+        fullName: 'IsDeleted (boolean)',
+        type: NodeType.MetadataField
+      }
+    ];
+    const getCmpsStub = stub(
+      MetadataOutlineProvider.prototype,
+      'getComponents'
+      ).returns(expected.map(n => n.fullName));
+    const customObjectmetadataObject = {
+      xmlName: 'customObject',
+      directoryName: 'testDirectory',
+      suffix: 'cls',
+      inFolder: false,
+      metaFile: false,
+      label: 'Custom Objects'
+    };
+    const parentNode = new BrowserNode(
+    'Custom Objects',
+    NodeType.MetadataType,
+    'CustomObject',
+    customObjectmetadataObject
+    );
+    parentNode.setComponents(['TestAccount', 'TestCleanInfo'], NodeType.Folder);
+    const childNodes = parentNode.children;
+    // @ts-ignore
```

Comment:
when suppressing/bypassing checks it would be good to specify what is being suppressed (the error details that is being ignored) and maybe also why

Reply:
agreed.  Looks like this has been removed but for future reference also big +1 to doc'ing why we need to tsignore if we do.

Reply:
Thanks @mohanraj-r setting a default value here, takes care of the ts error.